### PR TITLE
Write meta-description [YEP-79]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: tuneme/templates/_base.html [hunk 1]
```diff
@@ -11,10 +11,10 @@
         <link href='//fonts.googleapis.com/css?family=Quicksand:400,700' rel='stylesheet' type='text/css'>
         <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
         <meta http-equiv="content-type" content="text/html;charset=utf-8" />
-        <meta name="description" content="" />
-        <meta name="keywords" content="" />
+        <meta name="description" content="Tune Me puts the power in your hands to make choices about your body, your rights, your love life and your sexual health" />
+        <meta name="keywords" content="Zambia, love, sex, relationships, advice, help, family planning, contraception, private" />
         <meta property="og:title" content="" />
```
Comment (smarzban) [inline tuneme/templates/_base.html]: @bruskiza @miltontony what should I write for this one?

**Comment by smarzban** [comment]
@bruskiza @miltontony ready for review

**Comment by bruskiza** [comment]
:+1:

**Comment by smarzban** [comment]
@bruskiza @miltontony ready for the final review :)

**Comment by bruskiza** [comment]
:+1:

**Pull request merged**